docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+services:
+  monero-lws:
+    # depends_on:
+    #   - monero
+    container_name: monero-lws
+    build:
+      context: .
+    restart: always
+    ports:
+      - 8443:8443
+    command:
+      - --db-path=/home/monero-lws/.bitmonero/light_wallet_server
+      - --daemon=tcp://monero:1882
+      - --sub=tcp://monero:18084
+      - --zmq-pub=tcp://monero:18084
+      - --log-level=4
+      - --webhook-ssl-verification=none
+      - --disable-admin-auth
+      - --admin-rest-server=http://0.0.0.0:8443/admin
```

Comment:
The default should be `https` or port `8080`. I would advocate for `https`, but transient certificates are a kludge.